sql/core/src/test/resources/sql-tests/inputs/cast.sql

```diff
@@ -58,8 +58,10 @@ DESC FUNCTION EXTENDED boolean;
 -- TODO: migrate all cast tests here.
 
 -- cast string to interval and interval to string
-SELECT CAST('interval 3 month 1 hour' AS interval);
```

Comment:
do we still support `CAST('INTERVAL 1 YEAR 3 MONTH' AS INTERVAL)`?

Reply:
Can we add this test? to make sure people can still use mixed fields if they explicitly use the old interval type.